liquid_node/commands.py

```diff
@@ -450,6 +474,110 @@ def deletecollection(name):
     purge_collection(name)
 
 
+def importcollection(name, path, method='copy'):
+    """Import a single collection extracted collection.
+    The path must contain:
+    1. The blobs folder, named `blobs`
+    2. The database folder, named `pg`, containing a subfolder called `data`.
+    3. the index, named <name>-index.tgz
+
+    :param name: name of the collection you want to import
+    :param path: path to the collection data.
+    :param method: can be 'copy', 'move' or  'link'. Copy is default.
+    """
+    path = Path(path).resolve()
+    database = path / 'pg'
+    blobs = path / 'blobs'
+    index = path / f'{name}-index.tgz'
+    log.info(f'Importing collection {name}')
+    # parsing `method`, which is not interpreted as a kwarg
+    if method != 'copy':
+        _, method = method.split('=', 1)
+        if method not in ['copy', 'move', 'link']:
+            raise RuntimeError(f'unknown method: {method}')
+
+    node_name = name.lower()
+    if name != node_name:
+        log.info(f'Renaming "{name}" to "{node_name}"')
+    collection_path = Path(config.liquid_volumes) / 'collections' / node_name
+    if collection_path.exists():
+        if confirm(f'collection {name} already exists, overwrite (hoover needs to be running)?'):
+            redeploy = True
+            purge_collection(name)
+        else:
+            log.info(f'exiting and not overwriting data for {name}')
+            exit()
+    else:
+        redeploy = False
+        (collection_path).mkdir(parents=True)
+
+    # copy the pg dir
+    database = Path(database).resolve(strict=True)
+    if not (database / 'data' / 'PG_VERSION').exists():
+        raise RuntimeError("database is not a valid Postgres database")
```

Comment:
The normal way of exporting and importing a postgres database is to use  `pg_dump -Ox $dbname` which generates a text file. You can pipe the output through `gzip  -1` for a low-cpu effort to compress the data. I don't think we should be passing on-disk postgres database files around.

Reply:
This is the way it was done in `docker-setup` as well. Wasn't there a reason for doing like this?^^ 
I'll try whether exporting works with `pgdump` with the testdata set.

Reply:
there's a new branch `pg_dump` where my progress on changing this is in. The export works but the import doesn't yet. Unfortunately I have no time to fix this so I reverted the changes.